packages/frontend-2/components/form/Button.vue

```diff
@@ -104,31 +105,44 @@ const colorClasses = computed(() => {
   switch (props.type) {
     case 'outline':
       return `${
-        isDisabled ? 'text-foreground-3' : 'text-foreground'
-      } bg-transparent border-foreground-3 focus:ring-primary-lighter`
+        isDisabled
+          ? 'bg-neutral-300 dark:bg-neutral-800 text-neutral-100 dark:text-neutral-400 focus:ring-0 hover:ring-0'
```

Comment:
1. Is it OK that all disabled states are styled the same between all button types? How will they differ when put next to each other?
2. Should the colors used here not be added as semantic colors?

Reply:
TODO: add **disabled** semantic colors